deploy/production/deployment.tpl.yml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ${KUBE_NAMESPACE}
+  namespace: ${KUBE_NAMESPACE}
+  labels:
+    app: ${KUBE_NAMESPACE}
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 100%
+  selector:
+    matchLabels:
+      app: ${KUBE_NAMESPACE}
+  template:
+    metadata:
+      labels:
+        app: ${KUBE_NAMESPACE}
+    spec:
+      volumes:
+        - name: uploads
+          emptyDir: { }
+        - name: php-socket
+          emptyDir: { }
+      terminationGracePeriodSeconds: 35
+      serviceAccountName: ${KUBE_NAMESPACE}-service
+      containers:
+        - name: nginx
+          image: ${ECR_URL}:${IMAGE_TAG_NGINX}
+          ports:
+            - containerPort: 8080
+          volumeMounts:
+            - name: uploads
+              mountPath: /var/www/html/public/app/uploads
+            - name: php-socket
+              mountPath: /sock
+          resources:
+            requests:
+              cpu: 500m
+              memory: 200Mi
+            limits:
+              cpu: 500m
+              memory: 200Mi
+
+        - name: fpm
+          image: "${ECR_URL}:${IMAGE_TAG_FPM}"
+          ports:
+            - containerPort: 9000
```

Comment:
We don't need this anymore - sockets are king

Reply:
Good point 🤣